ereolen_support/js/hideOptions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file hideOptions.js
+ *   Hide certain select options in certain situations.
+ */
+(function () {
+  Drupal.behaviors.hideOptions = {
+    attach: function (context, settings) {
+      'use strict';
+
+      let problem = document.getElementById("edit-ereolen-support-problem");
+      hideOptions(problem);
+
+      problem.addEventListener('change', function(){
+        hideOptions(problem);
+      });
+    }
+  };
+
+  /**
+   * Hide options
+   */
+  function hideOptions(problem) {
+    let productOptions = document.getElementById("edit-ereolen-support-product").getElementsByTagName("option");
+    if (problem.value === 'Download') {
```

Comment:
Can the value "Download" be changed by translations in the backend?

Reply:
If so maybe `Drupal.t()` should be used?

Reply:
It can't be translated